common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala

```diff
@@ -0,0 +1,14 @@
+package com.gu.mediaservice.lib.collections
+
+import com.gu.mediaservice.model.Collection
+
+object CollectionsManager {
+  def onlyLatest(collections: List[Collection]): List[Collection] = {
```

Comment:
Any reason to not make `actionData` a list too? So that we store:

```json
{
    "collections": [
        {
            "pathId": "g2",
            "path": [
                "g2"
            ],
            "actionData": [{
                "author": "test",
                "date": "2015-11-23T10: 48: 59.475+00: 00"
              }, {
                "author": "test",
                "date": "2015-11-23T11: 28: 24.743+00: 00"
            }]
        },
        {
            "pathId": "g2/24%2F7",
            "path": [
                "g2",
                "24/7"
            ],
            "actionData": [{
                "author": "test",
                "date": "2015-11-23T10: 57: 48.505+00: 00"
            }]
        }
    ]
}
```

The logic to create/update would become more complex, but might be worth considering...